### PR TITLE
[JCS-1] Fetching Data from API

### PR DESCRIPTION
**Pull request by h-muhammad-ali**

### CODE REVIEW

**Comment by mali-ai** [comment]
@HafizMBilal009

File: main.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+async function getUsers() {
```
Comment (iambilalriaz) [inline main.js]: use arrow function syntax

File: main.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  let url = "https://jsonplaceholder.typicode.com/users";
+  try {
+    let result = await fetch(url);
+    return await result.json();
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+async function getUsersPosts() {
```
Comment (iambilalriaz) [inline main.js]: use arrow function syntax

File: main.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  let url = "https://jsonplaceholder.typicode.com/posts";
+  try {
+    let result = await fetch(url);
+    return await result.json();
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+async function getUserPostsById(id) {
```
Comment (iambilalriaz) [inline main.js]: use arrow function syntax

File: main.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        html +
+        `<pre>Post ID: ${post.id}     User ID: ${post.userId}</pre><p>Post Title: ${post.title}</p><p>Post Body: ${post.body}</p><hr>`;
+    }
+  });
+  html = html + `</div>`;
+  let container = document.getElementById("user" + id);
+  container.innerHTML = html;
+}
+
+async function displayUsers() {
```
Comment (iambilalriaz) [inline main.js]: use arrow function syntax

**Review by iambilalriaz — CHANGES_REQUESTED** [review]

**Review by iambilalriaz — CHANGES_REQUESTED** [review]
use `const` instead of `let` because function's definition is not being modified.

**Review by iambilalriaz — APPROVED** [review]

**Pull request merged**